### PR TITLE
Update GitHub issue templates

### PR DESCRIPTION
**Pull request by osanseviero**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bugs.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+**Bug description.**
+A clear and concise description of what the problem is. Ex. Clicking this button is not working when [...]
+
+**Describe the expected behaviour**
+A clear and concise description of what you want to happen.
+
+**Additional context**
+Add any other context or screenshots about the feature request here. Please share details such as browser when it's relevant.
```
Comment (pcuenca) [inline .github/ISSUE_TEMPLATE/bugs.md:20]:
```suggestion
Add any other relevant context or screenshots here. Please share details such as browser when appropriate.
```

**Review by pcuenca — APPROVED** [review]
Thank you!

File: .github/ISSUE_TEMPLATE/bugs.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug
+about: Report a bug you face in the Hub
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**This repository is focused on the Hub experience and documentation. If you're facing an issue with a specific library, please open an issue in the corresponding repo**
```
Comment (julien-c) [inline .github/ISSUE_TEMPLATE/bugs.md:10]:
```suggestion
**This repository is focused on the Hub experience and documentation. If you're facing an issue with a specific library, please open an issue in the corresponding GitHub repo. If you're facing an issue with a specific model or dataset, please open an issue in the corresponding HF repo.**
```

Comment (julien-c) [reply .github/ISSUE_TEMPLATE/bugs.md:10]: (maybe)

**Pull request merged**